fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/postaggregation/FieldAccessorPostAggregation.java

```diff
@@ -41,21 +41,40 @@ public FieldAccessorPostAggregation(@NotNull Aggregation aggregation) {
             throw new IllegalArgumentException(message);
         }
 
-        this.aggregation = aggregation;
+        this.metricField = aggregation;
```

Comment:
Shall we change `aggregation` to `metricField`?

Reply:
same applies here